specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/runHistory.json

```diff
@@ -0,0 +1,3042 @@
+            "format": "int32"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "description": "Whether to include a count of the matching resources along with the resources returned in the response.",
+            "required": false,
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Runs are returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedRunList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/history/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/runs:query": {
+      "post": {
+        "tags": [
+          "Run"
+        ],
+        "summary": "Get all Runs for a specific Experiment.",
+        "description": "Get all Runs for a specific Experiment with the specified query filters.",
+        "operationId": "Runs_GetByQuery",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "experimentName",
+            "in": "path",
+            "description": "The experiment name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "queryParams",
+            "in": "body",
+            "description": "Query parameters for data sorting and filtering.",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/QueryParams"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PaginatedRunList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/history/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/runcounts": {
```

Comment:
Is it too late to remove this API from our swaggers? Hadn't noticed this until this morning but this wasn't intended to be a public API

Reply:
Not too late.  It can be removed. @Greeshmamadalam

Reply:
sure I will remove it.

Reply:
done